orc-rt/CREDITS.TXT

```diff
@@ -0,0 +1,13 @@
+This file is a partial list of people who have contributed to the LLVM
+project. If you have contributed a patch or made some other contribution to
+LLVM, please submit a patch to this file to add yourself, and it will be
+done!
+
+The list is sorted by surname and formatted to allow easy grepping and
+beautification by scripts.  The fields are: name (N), email (E), web-address
+(W), PGP key ID and fingerprint (P), description (D), snail-mail address
+(S), and (I) IRC handle.
+
+N: Lang Hames
+E: lhames@gmail.com
+D: Initial code
```

Comment:
We've mostly stopped adding `CREDITS.TXT` files to new sub-projects... like 12 years ago. We probably don't need one here.